starlark-lsp/README.md

```diff
@@ -1,7 +1,10 @@
 # starlark lsp
 
-A language server for starlark based kurtosis packages, this is forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) and
-extended to support kurtosis builtin types and instructions. 
+It is a language server for Starlark with customized Kurtosis specific intellisense. You can learn more about language server here and learn more about Starlark here.
+
+
+## Acknowledgments
+This `starlark-lsp` is intially forked from [tilt-dev/starlark-lsp](https://github.com/tilt-dev/starlark-lsp) that contains Apache 2.0 liscense. It has been extended to support kurtosis builtins.
```

Comment:
s/is/was/g

s/that contains Apache 2.0 liscense/which is licensed as Apache 2.0 liscence/g 

s/kurtosis/Kurtosis